src/main/kotlin/com/terraformation/backend/seedbank/model/Accession.kt

```diff
@@ -237,6 +239,23 @@ data class AccessionModel(
                   runningRemaining
                 }
               }
+
+          // If the user withdrew all the seeds from a weight-based accession by entering the
+          // estimated seed count (which will be the case in, e.g., a nursery transfer which is
+          // always measured in seeds) then the remaining weight might be slightly more or less than
+          // zero thanks to imprecision of the subset weight measurement. We will end up with an
+          // observedMinusWithdrawals value less than the weight of one seed. Treat the withdrawal
+          // as having emptied out the accession in that case.
+          if (observedMinusWithdrawals.units != SeedQuantityUnits.Seeds &&
+              withdrawalsAfterObservation.isNotEmpty() &&
+              withdrawalsAfterObservation.last().withdrawn?.units == SeedQuantityUnits.Seeds &&
+              subsetCount != null &&
+              subsetWeightQuantity != null &&
+              observedMinusWithdrawals.abs() * subsetCount < subsetWeightQuantity) {
```

Comment:
This looks like it will round to zero if the remaining quantity (in seeds) is less than one subset, rather than one seed. Am I understanding that correctly?

Reply:
Multiplying by the subset count should make it compare the weight of one seed.

If it helps: originally I had this as
```
observedMinusWithdrawals.abs() < (subsetWeightQuantity / subsetCount)
```
but that was problematic because the division operation could produce very small values if the subset count was large, and they'd get rounded to zero and the comparison would fail. So I multiplied both sides by `subsetCount` to turn the division into a multiplication.

Reply:
Oh, that makes sense now 😄 Thanks!